docs/docs/how-tos/recursion-limit.ipynb

```diff
@@ -31,9 +31,26 @@
     "</div> \n",
     "\n",
     "\n",
-    "You can set the graph recursion limit when invoking or streaming the graph. The recursion limit sets the number of **supersteps** that the graph is allowed to execute before it raises an error. Read more about the concept of recursion limits [here](https://langchain-ai.github.io/langgraph/concepts/low_level/#recursion-limit). Let's see an example of this in a simple graph with parallel branches to better understand exactly how the recursion limit works.\n",
+    "You can set the graph recursion limit when invoking or streaming the graph. The recursion limit sets the number of [supersteps](../../concepts/low_level/#graphs) that the graph is allowed to execute before it raises an error. Read more about the concept of recursion limits [here](../../concepts/low_level/#recursion-limit). \n",
```

Comment:
should we actually add a sentence and a code snippet to demonstrate how to add a loop?
otherwise, the title says "how to create a loop" but we immediately jump to recursion limit